srcpkgs/libjxl/template

```diff
@@ -0,0 +1,78 @@
+# Template file for 'libjxl'
+pkgname=libjxl
+version=0.6.1
+revision=1
+#_short_version=0.6
+#wrksrc=libjxl-${_short_version}
```

Comment:
these comments should be removed before merging

Reply:
I have removed the two lines. They will likely be needed again though with the next major version (0.7).

Reply:
Why? I couldn't find a release tarball where the top directory doesn't match $pkgname-$version

Reply:
Sorry, it seems you're right, I don't know why I had a problem with one of the older versions.